locale/hu.json

```diff
@@ -4,14 +4,14 @@
         "humanizer_language": "hu"
     },
     "restarter": {
-        "start_timeout": "sikertelen indítás a megadott idő intervallumon belül",
-        "crash_detected": "összeomlás észlelve",
-        "hang_detected": "lefagyás észlelve",
+        "start_timeout": "Sikertelen indítás a megadott idő intervallumon belül",
+        "crash_detected": "Összeomlás észlelve",
+        "hang_detected": "Lefagyás észlelve",
```

Comment:
These 3 must be lower case, they are not phrases but inserted inside one.

Reply:
oh yes sorry this slid over my eye

Reply:
Sorry